bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
I can think of a few possible approaches here. I think your approach is buggy because of the many-to-one mapping.

Other possible approaches:

 - change all the clusters to include the Python type and the ZigBee type. This is probably the only correct way to do it.
 - Keep adding to the special list I've created on line 106+
 - Go over DATA_TYPES twice. In the first pass, put all the analog values in DATA_TYPE_IDX. In the second pass, only put discrete values in if there's no existing entry for that data type in DATA_TYPE_IDX already. This way would probably be my preference for trading off completeness and expedience.

Maybe there is a better way?